base_domain_operator/expression.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api
+# pylint: disable=W0402
+from openerp.osv.expression import ExtendedLeaf, expression, is_operator,\
```

Comment:
I think what @pedrobaeza means is that the linter should allow importing from `osv.expression` (but keep barking at stuff from osv)
We *could* also import expression from models I think (https://github.com/OCA/OCB/blob/8.0/openerp/models.py#L6241), but that's a bit convoluted in my opinion